### ISSUE TITLE
pxc requires socat

### ISSUE THREAD

**Issue by morgo** [datacharmer/dbdeployer#115]
**Describe the bug**

I tried to start a pxc cluster on a relatively fresh Ubuntu 20.04 machine (having already installed mysql etc). It is missing a library.

**To Reproduce**

```
$ dbdeployer deploy replication --topology=pxc 8.0.19
Installing and starting node 1
. sandbox server started
Installing and starting node 2
^C
```

It freezes on installing node 2. When I look at the error log, it complains about socat. I install it (`sudo apt install socat`), and everything works again :-)

**Expected behavior**

Similar to libaio, it should detect and complain.

**NO Screenshots**
If applicable, copy and paste the command and the output. Do not use screen shots, unless they add value to the report.  Text that can be edited is preferred to graphics.

**Environment:**
 - OS: Ubuntu 20.04
 - dbdeployer version: 1.53.2
 - tarball full name (e.g. mysql-5.6.40-linux-glibc2.12-x86_64.tar.gz): `Percona-XtraDB-Cluster_8.0.19-10.1_Linux.x86_64.focal.tar.gz`
 - tarball origin (from where it was downloaded) Percona directly today!

**Hardware: (if applicable)**
 - Free storage
 - Total RAM

**Additional context**

**Comment by datacharmer** [comment]
Thanks for this report.
I was working with incomplete information.
Apparently, `socat` was already installed in the server that I used to analyse PXC, and thus I have as prerequisites only `lsof` and `rsync`. The addition is just a line (`globals/globals.go`, line 480). I will fix it soon.

**Comment by datacharmer** [comment]
Fixed in [release 1.53.3](https://github.com/datacharmer/dbdeployer/releases/tag/v1.53.3)
